### DISCUSSION TITLE
How do I password protect the wifi hotspot?

### DISCUSSION THREAD

**Discussion by aksuhre** [iiab/iiab#3477] [Q&A] [answered]
Hi, I can't seem to figure out how to put a password on the iiab wifi. Is this possible? I am running it on raspberry pi 3b. 

I want to give access to the hotspot only to few people for a specific project.

Thanks in advance

**Comment by holta** [comment, marked as answer]
Hi @aksuhre,

[Internet-in-a-Box (IIAB)](https://internet-in-a-box.org) is generally intended for free libraries and schools with completely open access.

That said do you have command-line access to your IIAB using a screen/keyboard or ssh?  If so, can you try the following?

1. Run:
   ```
   sudo nano /etc/iiab/local_vars.yml
   ```
2. Look for [Lines 72-73](https://github.com/iiab/iiab/blob/f8efc93d0a2b1ac7cdcdb05d56d6a4b20c7fefd4/vars/local_vars_small.yml#L72-L73) in your [/etc/iiab/local_vars.yml](https://wiki.iiab.io/go/FAQ#What_is_local_vars.yml_and_how_do_I_customize_it%3F) and change them to be:
   ```
   hostapd_secure: True
   hostapd_password: ACTUAL_PASSWORD
   ```
3. Save the file (Ctrl-O) and exit (Ctrl-X).
4. Run the following, and confirm that it completes without error about 3 minutes later:
   ```
   sudo iiab-network
   ```
5. Test and please report back if this works?

**Reply by aksuhre** [reply]
Thanks @holta , this worked! Yay

**Reply by holta** [reply]
Posted to http://FAQ.IIAB.IO **> "21 Can I password protect the Wi-Fi hotspot?"**

**Comment by tim-moody** [comment]
This can also be done through Admin Console, though I haven't tried it recently.

**Reply by holta** [reply]
Where in Admin Console?  I didn't see that, but may be looking in the wrong place.
